### PR TITLE
Add known incoming link checker

### PR DESCRIPTION
**Pull request by paultyng**
This checks the top incoming links from Google Search Console to ensure there is still something at that destination. It filters out things we have explicit redirects for in our `redirects.txt`.

### CODE REVIEW

File: content/redirects.txt [hunk 1]
```diff
@@ -169,3 +169,6 @@
 # "configuration", because they relate to CLI functionality rather than the
 # configuration language.
 /docs/configuration/environment-variables.html /docs/commands/environment-variables.html
+
+# Load order and semantics is now partially covered by the "overrides" page for 0.12
+/docs/configuration/load.html /docs/configuration/override.html
```
Comment (paultyng) [inline content/redirects.txt]: Need to hear from someone on core about how to handle this redirect.

File: content/scripts/check-links.sh [hunk 2]
```diff
@@ -9,6 +9,26 @@ if [ -n "$REJECT_PATH" ]; then
 	REJECT_REGEX_ARG="--reject-regex ${URL}${REJECT_PATH}"
 fi
 
+redirects_file="$(mktemp)"
+cat content/redirects.txt| awk -F' ' '{print $1}' | grep -v '#' | sort | uniq > "$redirects_file"
```
Comment (paultyng) [inline content/scripts/check-links.sh]: This should probably be a more specific grep to only strip lines that start with `#`...

File: content/redirects.txt (hunk 1 above)
Comment (appilon) [reply content/redirects.txt]: Nitpick, missing NL?

File: content/scripts/testdata/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Check Links Test Data
+
+## Incoming Links
+
+These are downloaded from the [Google Search Console](https://search.google.com/u/0/search-console/links/drilldown?resource_id=sc-domain%3Aterraform.io&type=EXTERNAL&target=&domain=).
+
+The top 1000 can be opened in Google Sheets. Only links to `www.terraform.io` can be tested in this repository, make sure to filter out `registry.` or `app.` subdomain links.
+
+Strip the host and protocol from the urls and sorth alphabetically so the `incoming-links.txt` file looks like:
```
Comment (appilon) [inline content/scripts/testdata/README.md]: Nitpick, spelling on sort

**Comment by paultyng** [comment]
@radeksimko it failed due to a missing file in AWS, brian had fixed it but it wasn't in `stable-website` yet, just cherry picked and re-ran travis, now its breaking just in the normal crawl.

**Pull request merged**